src/io/ResultWriter.cpp

```diff
@@ -41,7 +41,8 @@ void ResultWriter::init(ParticleContainer * /*particleContainer*/,
 
 void ResultWriter::endStep(ParticleContainer * /*particleContainer*/, DomainDecompBase *domainDecomp, Domain *domain,
                            unsigned long simstep) {
-	map<unsigned, CavityEnsemble> * mcav = global_simulation->getMcav();
+	// moved to CavityWriter
```

Comment:
you can remove these lines completely. Note that the header needs to be changed in line 38 in this file!
Write a comment there, that the writing of cavities has been moved to the CavityWriter.

Reply:
Changed as requested ebeb2f0